Makefile

```
@@ -0,0 +1,155 @@
+POOL_SIZE ?= 300
+PLUGINS ?= maxim
+PROMETHEUS_LABELS ?= 
+
+# Colors for output
+RED=\033[0;31m
+GREEN=\033[0;32m
+YELLOW=\033[1;33m
+BLUE=\033[0;34m
+NC=\033[0m # No Color
+
+.PHONY: help dev build run install-air clean test ui-dev ui-build ui-install
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)